WordPress/gutenberg
# Combobox, FormTokenField: show message when no matches found

## What?


The PR allows the `ComboxControl` and `FormTokenField` components to show a message when there are no suggestions matching the input text.

## Why?


This PR is more of an enhancement and provides a better user experience when searching the `ComboboxControl` and `FormTokenField` components.

## How?


Displays a "No items found" message when no suggestions are found in the `SuggestionsList` component.

## Testing Instructions




Open Storybook and pick the Components > ComboboxControl component.  Search for an item that doesn't exist
and you should see `No items found` state.

## Screenshots or screencast 
![image](https://github.com/user-attachments/assets/d20ead18-846e-4e4a-ab3a-c1e79b4176ca)
